content/en/service_management/incident_management/incident_settings/information.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Information
+---
+
+## Overview
+
+From the [Incident Settings Information][1] page, define your organization’s severity levels and status levels, and declare incident helper text to reflect the specific needs of your organization. For example, you can customize severity settings differently for security and non-security incidents, creating a more nuanced response strategy. Or create custom helper text so that when an incident is declared, all the key pieces of information are added.
```

Comment:
```suggestion
From the [Incident Settings Information][1] page, define your organization’s severity levels and status levels, and declare incident helper text to reflect the specific needs of your organization. For example, you can customize severity settings differently for security and non-security incidents, creating a more nuanced response strategy. You can also create custom helper text so that when an incident is declared, all the key pieces of information are added.
```

Starting a sentence with "Or" has a marketing vibe. I'm not sure if I can explain further.